.taskcluster.yml

```diff
@@ -527,3 +527,39 @@ tasks:
           description: Update Taskcluster hook triggering the code-review integration tests
           owner: bastien@mozilla.com
           source: https://github.com/mozilla/code-review
+
+    - $if: 'tasks_for == "github-push" && head_branch[:10] == "refs/tags/"'
+      then:
+        $let:
+          version: {$eval: 'head_branch[10:]'}
+        in:
+          taskId: {$eval: as_slugid("release")}
+          dependencies:
```

Comment:
Maybe we should depend on the test tasks too, so we only release when everything is green

Reply:
The build steps already depend on the tests, so we should be fine